engine/src/main/java/com/sandboni/core/engine/finder/bcel/visitors/CallerFieldOrMethodVisitor.java

```diff
@@ -22,11 +19,13 @@
     CallerFieldOrMethodVisitor(Method m, JavaClass jc, Context c) {
         super(m, jc, c);
         cp = new ConstantPoolGen(jc.getConstantPool());
-        boolean testMethod = getAnnotation(jc.getConstantPool(), m::getAnnotationEntries, JUNIT_PACKAGE, TESTING_PACKAGE) != null;
 
-        boolean ignore = Objects.nonNull(AnnotationUtils.getAnnotation(jc.getConstantPool(), jc::getAnnotationEntries, Annotations.TEST.IGNORE.getDesc()));
+        boolean testMethod = getAnnotation(jc.getConstantPool(), m::getAnnotationEntries, ALL_TEST_PACKAGES).isPresent();
+
+        boolean ignore = AnnotationUtils.isIgnore(jc, jc::getAnnotationEntries);
```

Comment:
line 25+28 can be combined with || ('or')

Reply:
Good point, done!